### PR TITLE
feat: build AMIs using Github Actions

### PR DESCRIPTION
**Pull request by darora**
Builds ARM Postgres AMIs from Github Actions.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: [self-hosted, linux]
+    timeout-minutes: 150
+
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+
+      - name: Build AMI
+        run: |
+          packer build -timestamp-ui -color=false -on-error=abort -var "ansible_arguments=--tags,update-only,--tags,install-pgbouncer,--tags,install-supabase-internal" -var-file common.vars.json -var-file development-arm.vars.json amazon.json
```
Comment (darora) [inline .github/workflows/ci.yml]: @dragarcia do you wanna take this PR over and flesh out what this needs to be?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+name: Run CI checks
+
+on:
+  push:
+    branches:
+      - master
```
Comment (dragarcia) [inline .github/workflows/ci.yml]: will change this to `develop`. Will leave updates to `master` for external releases.

**Comment by darora** [comment]
fails on walg build now

**Comment by dragarcia** [comment]
Suggested changes via this [PR](https://github.com/supabase/postgres/pull/106). We can build an internal image from scratch with just one set of `ansible_arguments`. Will just be dependent on this [PR](https://github.com/supabase/postgres/pull/98) coming in first. 

> fails on walg build now

Resolved by the same [PR](https://github.com/supabase/postgres/pull/98) above

**Pull request merged**